release notes/v0.40.0.md

```diff
@@ -0,0 +1,280 @@
+
+# Breaking changes
+
+- [#2632](https://github.com/grafana/k6/pull/2632) During the refactoring to set tags to `metric.add` in the order they are provided, we discovered that you could provide tags as key-value pair map *multiple* times in the same call. This has never been the intended use and was never documented. As it was undocumented, and as such functionality makes no sense alongside every other API k6 has, we decided to remove this ability.
+- [#2582](https://github.com/grafana/k6/pull/2582) Previously, if `RegisterCallback` result was called twice, the second call would silently break the event loop. This has never been expected behavior, and calling it twice is always a bug in the code using it. Now, calling the `RegisterCallback` result twice leads to a panic.
+- [#2596](https://github.com/grafana/k6/pull/2596) The `tainted` property of the Metric type is no longer outputted by the JSON output. That property was likely always going to have a `false` value as it was outputted at the beginning of the test.
+
+## Main module/script no longer taints the global scope [#2571](https://github.com/grafana/k6/pull/2571)
+
+During the ESM changes, we found that anything defined in the main module scope was also accessible globally. This was because it was directly evaluated in the global scope.
+This has now been remedied and is no longer the case. This is a *breaking* change, but given that the whole point of modules (CommonJS or ESM) is to separate them, this is obviously rather a bug than a feature.
+
+On that note, we've seen [reports](https://github.com/grafana/k6/issues/2623) by people who have this global accessibility of the main module (intentionally or not). Still, it seems relatively rare, with only a few usages in a script. So if you need to access something globally, our suggested workaround is to set it explicitly on the global object `globalThis`.
+
+## `k6/ws` now considers the `throw` option [#2247](https://github.com/grafana/k6/issues/2247)
+
+`k6/http` has used the [`throw` option](https://k6.io/docs/using-k6/k6-options/reference/#throw) to figure out whether it should throw an exception on errors or return a response object with an error set on it (and log it).
+
+This functionality is finally also available for `k6/ws`, which previously would've *always* thrown an exception and thus involved more scripting to handle it.  [#2616](https://github.com/grafana/k6/pull/2616)
+
+This is a minor breaking change. By default, `throw` is `false`, so it now no longer throws an exception but instead returns a Response with `error` property.
+
+Thank you, @fatelei, for making this change !
+
+# New Features
+
```

Comment:
```suggestion
# New Features

Introducing experimental modules and first-class classes!

```

Reply:
I think it's nice to have little guiding text to improve the overall document flow.

Reply:
I am not so certain I prefer this ... 

WDYT @codebien @na-- @olegbespalov

Reply:
Same, we are already mentioning it at the top.
If we can add more valuable content there I agree that it could improve the flow but I don't see it in the current suggestion and I don't have a better proposal.